libs/watt/package/modal/+storybook/watt-modal-from-class.component.ts

```diff
@@ -37,18 +38,20 @@ import { FormControl, ReactiveFormsModule } from '@angular/forms';
     </watt-modal>
   `,
 })
-export class WattModalComponent extends WattTypedModal<string> implements OnInit {
-  title = 'This is a modal opened from a class';
+export class WattModalComponent extends WattTypedModal<string> {
+  title = signal('This is a modal opened from a class');
   usernameControl = new FormControl('');
   passwordControl = new FormControl('');
-  isLoading = false;
+  isLoading = signal(false);
 
-  ngOnInit(): void {
-    this.isLoading = true;
-
-    setTimeout(() => {
-      this.isLoading = false;
-    }, 300);
+  constructor() {
+    super();
+    effect(() => {
+      this.isLoading.set(true);
+      setTimeout(() => {
+        this.isLoading.set(false);
+      }, 300);
+    });
```

Comment:
nitpick: there's no need to put the signal update in an `effect`. The previous solution was just as fine :)